src/linear_algebra/basis.lean

```diff
@@ -1424,3 +1424,52 @@ let ⟨q, hq⟩ := p.exists_is_compl in nonempty.intro $
   (prod_equiv_of_is_compl q p hq.symm)
 
 end division_ring
+
+section restrict_scalars
+
+variables [comm_ring R] [ring K] [nontrivial K] [add_comm_group V]
```

Comment:
It's somewhat surprising that `K`/`V` are a ring and a module, not a field and a vector space. Could you change the variable names?